test/utility/mockApp.ts

```diff
@@ -2,8 +2,8 @@ import * as models from 'powerbi-models';
 
 export interface IApp {
   // Load
-  load(config: models.ILoadConfiguration): Promise<void>;
-  validateLoad(config: models.ILoadConfiguration): Promise<models.IError[]>;
+  load(config: models.IReportLoadConfiguration): Promise<void>;
+  validateLoad(config: models.IReportLoadConfiguration): Promise<models.IError[]>;
```

Comment:
add in general what's the difference between validate(config) above and validateLoad(config) here?